metrics/statsd_reporter.go

```diff
@@ -33,6 +33,7 @@ import (
 type Client interface {
 	Count(name string, value int64, tags []string, rate float64) error
 	Gauge(name string, value float64, tags []string, rate float64) error
+	SimpleEvent(title string, text string) error
```

Comment:
Why `SimpleEvent` and not just `Event`? Are there other types of events?

Reply:
It matches the interface from datadog-go package: https://github.com/DataDog/datadog-go/blob/master/statsd/statsd.go#L191